test/support/test_config.h

```diff
@@ -166,4 +166,11 @@
 // of the underlying sycl::device to determine whether to include or exclude 64-bit type tests.
 #define _PSTL_GROUP_REDUCTION_MULT_INT64_BROKEN 1
 
+// oneAPI DPC++ compiler in 2022.2 shows an internal compiler error during the backend code generation of minmax_element.pass.cpp
+#if TEST_DPCPP_BACKEND_PRESENT && (__INTEL_LLVM_COMPILER == 20220200 || __INTEL_LLVM_COMPILER == 20220201)

```

Comment:
Does this only fail specifically 2022.2.0 and 2022.2.1? (what about 2022.1 and before?)

Would we be better off checking `(__INTEL_LLVM_COMPILER >= 20220200 &__INTEL_LLVM_COMPILER < 20220300)` to cover future possible hotfixes to 2022.2.x?  I'd expect they wouldn't fix this specific issue in a hotfix, but I'm not sure what the right approach is here.

Reply:
It looks like older compilers are affected as well. I've changed this to check for releases before 2022.3.